### DISCUSSION TITLE
Laptop Keyboard light does'n work

### DISCUSSION THREAD

**Discussion by IlyaDin** [seerge/g-helper#2493] [General]
Hello! I am sorry for my English.
Your application is absolutely incomparable!
I have a laptop Asus ROG Strix G814JVR
![Uploading Screenshot 2024-04-25 152728.png…]()
I'm having trouble getting the keyboard backlight to work.
Keyboard backlight animation effects work during loading.
During operation, when switching lighting styles, nothing changes, one color is always on.

**Comment by IlyaDin** [comment]
<img width="689" alt="Screenshot 2024-04-25 152728" src="https://github.com/seerge/g-helper/assets/53399200/6fdb33e4-70bb-4806-b43d-d8748149aa38">

**Comment by seerge** [comment]
@IlyaDin your questions are already addressed in FAQ 
https://github.com/seerge/g-helper/wiki/FAQ#backlight-doesnt-change-color-or-stays-blue

**Reply by IlyaDin** [reply]
**Thank you** very much for your time.
I'm stupid, I searched using keywords "keyboard" and "Visual Mode"
Unfortunately "Stripe" and "PayPal" do not work in my country (Russia).
This whole situation...
**Thank you very much!**
